tests/Jobs/RockTheVote/ImportRockTheVoteReportTest.php

```diff
@@ -13,11 +13,11 @@
 class ImportRockTheVoteReportTest extends TestCase
 {
     /**
-     * Test that report is not downloaded when status not complete.
+     * Test that report is not downloaded and job is retried if status is building.
```

Comment:
I thought it was only supposed to be retried if it failed?

Reply:
Ahh this is a confusing comment on my end. We dispatch a job to check this report status again after 2 minutes since it's still building. Maybe this should just read as 
```
Test that a job is dispatched to import this report again after 2 minutes if its status is building
```

Reply:
I like this alternative!